docs/user_guide/connecting/amazon-rdb.md

```diff
@@ -0,0 +1,31 @@
+Connecting to Amazon RDB requires you to set up an IAM user and ensure your security group allows traffic from your IP address.
+
+## IAM Configuration in Amazon
```

Comment:
Questions:

- [ ] This is just for postgres, no?
- [ ] Does this require them to have a credential file locally? If so, needs mentioning, and a link providing to how to set that up
- [ ] Are there any links you can provide to AWS docs for how to do some of this stuff, eg setting up an IAM user, opening ports in security groups?

Reply:
This has only be done on Postgres, However, if you give me one more week, I can see about Aurua DB, Maria DB, and MSQL Server or can do it as another PR.

This does still require credential file locally - updated the docs

Updated the docs to include relevant links.